src/controllers/VariableController.ts

```diff
@@ -228,13 +228,23 @@ export class VariableController {
         return res.setVariableIsReadonly(id, isReadonly).then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method gets the image variable connector id.
+     * @param id The id of the image variable
+     * @returns The id of the connector
+     */
+    getImageVariableConnectorId = async (id: string) => {
+        const res = await this.#editorAPI;
+        return res.getImageVariableConnectorId(id).then((result) => getEditorResponseData<Id>(result));
+    };
+
     /**
      * This method sets the image variable connector. Setting a connector will
      * automatically remove the assetId linked to the connector if present.
      * If a connector was the source of the variable, it will be unregistered.
-     * @param id The ID of the image variable to update
-     * @param registration registration object containing all details about the connector
-     * @returns
+     * @param id The id of the image variable to update
+     * @param connectorId The new id of the connector
```

Comment:
The docs are updated, bur the method still accepts a `ConnectorRegistration` object? I believe this should also be updated to pas an id?

Reply:
Thanks for noticing this, probably a bad conflict resolution of me.

(it's actually the other way around)